parachains/runtimes/assets/asset-hub-polkadot/src/lib.rs

```diff
@@ -761,6 +767,9 @@ construct_runtime!(
 		Uniques: pallet_uniques::{Pallet, Call, Storage, Event<T>} = 51,
 		Nfts: pallet_nfts::{Pallet, Call, Storage, Event<T>} = 52,
 		ForeignAssets: pallet_assets::<Instance2>::{Pallet, Call, Storage, Event<T>} = 53,
+
+		// Storage aux.
+		PagedList: pallet_paged_list::{Pallet} = 80,
```

Comment:
Following the idiom of bags-list, let's name this something better, such that if it is used for a second time, the names don't collide?